packages/backports/installer/cos.sh

```diff
@@ -266,10 +266,10 @@ usage()
     echo ""
     echo "Example: $PROG-install /dev/vda"
     echo "  install:"
-    echo "  [--partition-layout /path/to/config/file.yaml ] [--force-efi] [--force-gpt] [--docker-image IMAGE] [--no-verify] [--no-cosign] [--iso https://.../OS.iso] [--debug] [--tty TTY] [--poweroff] [--no-format] [--config https://.../config.yaml] DEVICE"
+    echo "  [--partition-layout /path/to/config/file.yaml ] [--force-efi] [--force-gpt] [--docker-image IMAGE] [--verify] [--no-cosign] [--iso https://.../OS.iso] [--debug] [--tty TTY] [--poweroff] [--no-format] [--config https://.../config.yaml] DEVICE"
     echo ""
     echo "  upgrade:"
-    echo "  [--strict] [--recovery] [--no-verify] [--no-cosign] [--directory] [--docker-image] (IMAGE/DIRECTORY)"
```

Comment:
this is the previous installer backport, I'd leave it as it is, as it serves just for legacy support

Reply:
ok, let's leave it